src/taxes/views/TaxClassesList.tsx

```diff
@@ -0,0 +1,40 @@
+import useNavigator from "@saleor/hooks/useNavigator";
+import React from "react";
+
+import { taxClasses } from "../fixtures";
+import TaxClassesPage from "../pages/TaxClassesPage";
+import { taxClassesListUrl, taxTabSectionUrl } from "../urls";
+import { useTaxUrlRedirect } from "../utils/useTaxUrlRedirect";
+
+interface TaxClassesListProps {
+  id: string | undefined;
+}
+
+export const TaxClassesList: React.FC<TaxClassesListProps> = ({ id }) => {
+  const navigate = useNavigator();
+
+  const handleTabChange = (tab: string) => {
+    navigate(taxTabSectionUrl(tab));
+  };
+
+  useTaxUrlRedirect({
+    id,
+    data: taxClasses,
+    navigate,
+    urlFunction: taxClassesListUrl
+  });
+
+  if (id === "undefined" && taxClasses) {
```

Comment:
Not sure I understand the latter part of this condition, why make a check for a fixture? Won't it always be true?

Reply:
It's a fixture for now, when I change this to API data I will have to change less stuff.

Reply:
Ah, okay then